bukkit/src/main/java/us/myles/ViaVersion/bukkit/providers/BukkitInventoryQuickMoveProvider.java

```diff
@@ -26,8 +26,8 @@
 
 public class BukkitInventoryQuickMoveProvider extends InventoryQuickMoveProvider {
 
-    private static Map<UUID, BukkitInventoryUpdateTask> updateTasks = new ConcurrentHashMap<UUID, BukkitInventoryUpdateTask>();
-    private boolean supported;
+    private static final Map<UUID, BukkitInventoryUpdateTask> UPDATE_TASK = new ConcurrentHashMap<>();
```

Comment:
UPDATE_TASKS?

Reply:
just changed it according to constant naming conventions. if you say it looks cleaner in lower camel, I can just change it back 👀

Reply:
waiit, not even sure why it's static, I'll check that in a minute